react/features/stream-effects/virtual-background/JitsiStreamBackgroundEffect.ts

```diff
@@ -104,16 +111,21 @@ export default class JitsiStreamBackgroundEffect {
             this._inputVideoElement.width,
             this._inputVideoElement.height
         );
-        this._outputCanvasCtx.globalCompositeOperation = 'source-in';
-        this._outputCanvasCtx.filter = 'none';
+        if (this._outputCanvasCtx) {
+            this._outputCanvasCtx.globalCompositeOperation = 'source-in';
+            this._outputCanvasCtx.filter = 'none';
+        }
 
         // Draw the foreground video.
-        this._outputCanvasCtx.drawImage(this._inputVideoElement, 0, 0);
+        // @ts-ignore
+        this._outputCanvasCtx?.drawImage(this._inputVideoElement, 0, 0);
 
         // Draw the background.
-        this._outputCanvasCtx.globalCompositeOperation = 'destination-over';
+        if (this._outputCanvasCtx) {
```

Comment:
you could avoid all these checks with a quick return if `_outputCanvasCtx` is falsy